tron/core/actionrun.py

```diff
@@ -442,7 +454,27 @@ def fail(self, exit_status=0):
 
         return self._done('fail', exit_status)
 
+    def emit_triggers(self):
+        if isinstance(self.trigger_downstreams, bool):
+            shortdate = self.render_template("%(shortdate)s")
+            triggers = [f"shortdate.{shortdate}"]
+        elif isinstance(self.trigger_downstreams, dict):
+            rendered = [
+                (k, self.render_template(v))
+                for k, v in self.trigger_downstreams.items()
+            ]
+            triggers = [f"{key}.{value}" for key, value in rendered]
+        else:
+            log.error(f"{self} trigger_downstreams must be true or dict")
```

Comment:
Do we need to worry about this error here? trigger_downstreams should be validated through the config parser already, right?

Reply:
we don't *need* to, but it doesn't hurt to be a bit more explicit when something changes in future